index.bs

```diff
@@ -17,7 +17,10 @@ Abstract: This specification defines several new DOM events that provide informa
 Boilerplate: omit issues-index, omit conformance, repository-issue-tracking no
 Include MDN Panels: if possible
 Issue Tracking: DeviceOrientation Event Specification Issues Repository https://github.com/w3c/deviceorientation/issues
+Status Text: <p>This specification was initially published as a <a href="https://www.w3.org/TR/2016/CR-orientation-event-20160818/">Candidate Recommendation</a> in August 2016 and was <a href="https://www.w3.org/TR/2017/NOTE-orientation-event-20170530/">retired</a> in 2017 due to the Geolocation Working Group closure. In 2019 Devices and Sensors Working Group adopted this specification and during 2019-2024 made substantial interoperability, test automation, privacy and editorial improvements as outlined in the <a href="#changes">Changes</a> section. These changes aligned the specification with widely available implementations. In 2024 this specification became a joint deliverable between the Devices and Sensors Working Group and WebApps Working Group.</p><p>This document is maintained and updated at any time. Further changes are expected to be reflected in revised Candidate Recommendation Drafts and Snaphots as deemed appropriate to align the specification with existing implementations. Before requesting transition to Proposed Recommendation, the Working Groups will seek to demonstrate there is at least two independent, interoperable implementations of every feature defined in the specification, where interoperability can be verified by passing open test suites, and two or more implementations interoperate with each other.</p>
```

Comment:
Did you mean "two or more implementations interoperat**ing** with each other"?

Reply:
Yes indeed, fixed in https://github.com/w3c/deviceorientation/pull/142/commits/ce6740b6c44cfa8395213478750ca52a49a3e36a